app/plugin_test.go

```diff
@@ -82,6 +102,10 @@ func TestPluginKeyValueStore(t *testing.T) {
 	list, err = th.App.ListPluginKeys(pluginId, 2, 2)
 	assert.Nil(t, err)
 	assert.Equal(t, 0, len(list))
+
+	// Clean up
+	assert.Nil(t, th.App.DeletePluginKey(pluginId, "key"))
```

Comment:
This is orthogonal to this change, but if a panic or `t.Fatal` occurs above, the test won't clean up, and we won't necessarily have a clean database. I'm addressing this kind of issue in a parallel change, but I would suggest at least moving this to the top of the function as a `defer()` statement.

Reply:
Done.